package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "wdio-wikibase",
-	"version": "5.1.0",
+	"version": "5.2.0",
```

Comment:
If the API changes incompatibly (module now exports a class instead of an instance), shouldn’t this bump the major version number (6.0.0)?

Reply:
Yeah, that makes sense.